### ISSUE TITLE
初始版本

### ISSUE THREAD

**Issue by Xanliang** [Xanliang/VXI11.CSharp#1]
希望大家共同完善

**Comment by ATECoder** [comment]
I am unable to communicate with a Keithley 7510 instrument when running the TestClient in debug or executable.   
When sending the *IDN?, the instrument reports two errors:
-363 input buffer overrun: too many characters were sent in one line.
-420 Query Unterminated (from VXI-11): the instrument was asked to send information but there was nothing to send,

thereafter the test client times out with Read Error Code 15

The TestClient works okay with the TestServer returning 'PhaseLock Technology,EXXX0A'

I am able to send the *IDN? command using other programs.

It seems to me that the instrument is not receiving the commands. To test this, I sent the "SYST.CLE" command, which should have cleared the error log but nothing happened.  This would explain the query unterminated error, I verified ending the command with the new line character and even added a carriage return (CRLF is acceptable for this instrument). I also tried adding an extra new line character.

Environment:
Visual Studio Community 2022 edition current version 17.4.326.54890
.NET: 4.8

**Comment by Xanliang** [comment]
@ATECoder   do you resolve this problem？

**Comment by ATECoder** [comment]
Yes, thank you. There was an issue with the Flags, which I discovered using Wireshark and comparing with Keysight VISA.
